packages/shared/features/features.ts

```diff
@@ -101,6 +101,11 @@ const features = {
         sendAndReceive: {
             enabled: true,
         },
+        send: {
+            nft: {
+                enabled: false,
+            },
+        },
```

Comment:
Maybe this can go under `sendAndReceive`?
```suggestion
        sendAndReceive: {
            enabled: true,
             nft: {
                enabled: false,
            },
```